docs/_pages/docs/03-understand/05-trade-escrow.md

```diff
@@ -2,10 +2,60 @@
 sidebar:
   title: '<img id="side-icon-verybig" src="/assets/vector/money-bill-transfer.svg"/>Escrow'
   nav: docs
 src: "_pages/docs/03-understand/05-trade-escrow.md"
 ---
 
-{% include wip %}
\ No newline at end of file
+When selling bitcoin, a trade escrow is used to protect the buyer from fraud or nonpayment. The escrow acts as an assurance of security, leveraging Lightning [hold invoices](https://github.com/lightningnetwork/lnd/pull/2022) for a trustless transaction between robots.
+
+The alotted time to submit a trade escrow is determined by the order maker. The escrow's expiry timer defaults to three hours; however, this can be customized to range anywhere from one to eight hours.
+
+If the seller fails to submit the trade escrow within the order's given time limit, then the seller forfeits their fidelity bond. As well, if a dispute is opened, the satoshis in escrow are sent to the dispute winner.
+
+Be sure to use a Lightning wallet that plays well with RoboSats, refer to [Understand > Wallets](https://learn.robosats.com/docs/wallets/) for additional information.
+
+*Note: The term "seller" specifically refers to selling bitcoin while "buyer" specifically refers to buying bitcoin.*
+
+## **How to Submit a Trade Escrow**
+
+First, refer to [Understand > Wallets](https://learn.robosats.com/docs/wallets/) for compatible Lightning wallets that will help make using RoboSats a smoother experience. Depending on the wallet, the locked funds might show as a payment that is in transit, frozen, or even appearing to fail. Check the wallet compatability list!
+
+Read the relevant guide depending on if you are making or taking an order to sell bitcoin:
+* **Maker**: Select "Make Order" and modify the order conditions to your liking. The order can be customized to require an "Escrow Deposit Time-Out" (expiry timer) other than the default of three hours, ranging anywhere from one to eight hours. When your live order gets taken and the taker submits their payout invoice, use the shown QR code found in the "Contract Box" with your Lightning wallet to lock the indicated amount of sats as collateral (escrow). You can now chat with your robot peer to coordinate the fiat payment. *Note: Escrow funds are released to the buyer once you select "Confirm Fiat Received" which settles the order. Only confirm after the fiat has arrived in your account.*
+* **Taker**: Browse the order book and find an order to your liking. Simply select the "Take Order" option and lock your fidelity bond. Only after the buyer submits their payout invoice will you be required to submit the trade escrow. Use the shown QR code found in the "Contract Box" with your Lightning wallet to lock the indicated amount of sats as collateral (escrow). You can now chat with your robot peer to coordinate the fiat payment. *Note: Escrow funds are released to the buyer once you select "Confirm Fiat Received" which settles the order. Only confirm after the fiat has arrived in your account.*
```

Comment:
As already noted by you in the Dev matrix group, locking the escrow / submitting invoice happens simultaneously.

Takers that are sellers will always have to lock two hodl invoices back-to-back. First the fidelity bond and just after that the escrow. Sometimes this might be misleading because some users do not realize the QR code and the prompt has changed after locking the bond :sweat_smile: